nflow-engine/src/main/resources/scripts/db/update-7.1.0-x/sqlserver.update.ddl.sql

```diff
@@ -1,5 +1,7 @@
 drop index nflow_archive_workflow_uniq;
 
+drop index nflow_archive_workflow_parent;
```

Comment:
If we drop the index on update, should we also remove the index creation from the create `sqlserver.create.ddl.sql`?